cmd/clusterctl/client/init.go

```diff
@@ -73,6 +73,9 @@ type InitOptions struct {
 	// IgnoreValidationErrors allows for skipping the validation of provider installs.
 	// NOTE this should only be used for development
 	IgnoreValidationErrors bool
+
+	// allowMissingProviderCRD allows listing of images if the provider CRD is missing.
+	allowMissingProviderCRD bool
```

Comment:
Is there a particular reason on why we'd need this flag? Could we just always return the images regardless if the CRDs are installed or not?

Reply:
That would be good context to provide in the flag, wdyt?

Reply:
Sounds good. @aniruddha2000 Can you please update the flag description?

Reply:
Sure 😄